CONTRIBUTING.md

```diff
@@ -2,52 +2,95 @@
 
 ## Code of Conduct
 
-All interactions with this project follow our [Code of Conduct][code-of-conduct].
-By participating, you are expected to honor this code. Violators can be banned
-from further participation in this project, or potentially all Coinbase projects.
+All interactions with this project follow our [Code of Conduct](https://github.com/coinbase/code-of-conduct). By participating, you are expected to honor this code. Violators can be banned from further participation in this project, or potentially all Coinbase projects.
 
-[code-of-conduct]: https://github.com/coinbase/code-of-conduct
+## How to Contribute
 
-## Bug Reports
+You can contribute to this repository by asking questions, providing feedback, and reporting issues.
 
-* Ensure your issue [has not already been reported][1]. It may already be fixed!
-* Include the steps you carried out to produce the problem.
-* Include the behavior you observed along with the behavior you expected, and
-  why you expected it.
-* Include any relevant stack traces or debugging output.
+### Asking Questions
 
-## Feature Requests
+Submit your questions via the [Rosetta Community boards][13].
 
-We welcome feedback with or without pull requests. If you have an idea for how
-to improve the project, great! All we ask is that you take the time to write a
-clear and concise explanation of what need you are trying to solve. If you have
-thoughts on _how_ it can be solved, include those too!
+### Providing Feedback
+
+You can also use the [Rosetta Community boards][13] to provide feedback.
+
+### Reporting Issues
+
+You can report issues by submitting bug reports, feature requests, or pull requests via GitHub. You **must** submit [security issues](#security-issues) and [support requests](#support-requests) through the links provided.
+
+#### Bug Reports
+
+Before filing a bug report, ensure that your issue [has not already been reported][1]. It may already be fixed!
+
+If your bug hasn’t been fixed, follow these steps to file a bug report:
+
+1. [Open an issue in GitHub][10].
+2. Add a title for your bug report. It should briefly describe the problem.
+3. Follow the template that appears in the Write text box. This is the best way to describe the bug.
+    * **Describe the bug** - A clear and concise description of what the bug is.
+    * **To Reproduce** - Steps to reproduce the behavior.
+    * **Expected behavior** - A clear and concise description of what you expected to happen.
+    * **Additional context** - You can include any relevant stack traces or debugging output in this section.
```

Comment:
Why do we need to mention this here, if we can mention that in issue template?  It will be redundant and will diverge all the time if we change anything in Issue template.
You can put the detailed explanation as comments in the template so that the author knows about them.

Reply:
That works. I can remove and add the additional info to the template.

Reply:
I didn't see any additional info to add to the template so I did't make any changes there.